CHANGELOG.md

```diff
@@ -29,7 +29,8 @@
   * Ruby 2.4.10, 2.5.8, 2.6.6, 2.7.1 [\#4904](https://github.com/rvm/rvm/issues/4904)
   * Ruby 2.4.8 [\#4916](https://github.com/rvm/rvm/issues/4916)
   * Ruby 2.1.0-2.1.10, 2.2.0-2.2.9, 2.3.0-2.3.6, 2.4.0-2.4.3 [\#4916](https://github.com/rvm/rvm/issues/4916)  
-
+* Ubuntu 14.04 x64 binaries
+  * Ruby 2.1.0-2.7.1
```

Comment:
Could you please provide more specific ranges, as I did here: https://github.com/rvm/rvm/pull/4934/files#diff-4ac32a78649ca5bdd8e0ba38b7006a1eR28

Reply:
Done